package/kedro_viz/data_access/repositories/modular_pipelines.py

```diff
@@ -1,241 +1,241 @@
-        """
-        modular_pipeline = self.get_or_create_modular_pipeline(modular_pipeline_id)
-        modular_pipeline.children.add(child)
-
-    def extract_from_node(self, node: GraphNode) -> Optional[str]:
-        """Extract the namespace from a graph node and add it as a modular pipeline node
-        to the modular pipeline repository.
-
-        Args:
-            node: The GraphNode from which to extract modular pipeline.
-        Returns:
-            ID of the modular pipeline node added to the modular pipeline repository if found.
-        Example:
-            >>> modular_pipelines = ModularPipelinesRepository()
-            >>> model_output_node = GraphNode.create_data_node(
-            ...     "data_science.model_output", layer=None, tags=set(), dataset=None
-            ... )
-            >>> modular_pipelines.extract_from_node(model_output_node)
-            'data_science'
-            >>> assert modular_pipelines.has_modular_pipeline("data_science")
-        """
-
-        # There is no need to extract modular pipeline from parameters
-        # because all valid modular pipelines are encoded in either a TaskNode or DataNode.
-        if isinstance(node, ParametersNode):
-            return None
-
-        modular_pipeline_id = node.namespace
-        if not modular_pipeline_id:
-            return None
-
-        modular_pipeline = self.get_or_create_modular_pipeline(modular_pipeline_id)
-
-        # Add the node's registered pipelines to the modular pipeline's registered pipelines.
-        # Basically this means if the node belongs to the "__default__" pipeline, for example,
-        # so does the modular pipeline.
-        modular_pipeline.pipelines.update(node.pipelines)
-
-        # Since we extract the modular pipeline from the node's namespace,
-        # the node is by definition a child of the modular pipeline.
-        self.add_child(
-            modular_pipeline_id,
-            ModularPipelineChild(id=node.id, type=GraphNodeType(node.type)),
-        )
-        return modular_pipeline_id
-
-    def has_modular_pipeline(self, modular_pipeline_id: str) -> bool:
-        """Return whether this modular pipeline repository has a given modular pipeline ID.
-        Args:
-            modular_pipeline_id: ID of the modular pipeline to check existence in the repository.
-        Returns:
-            Whether the given modular pipeline ID is in the repository.
-        Example:
-            >>> modular_pipelines = ModularPipelinesRepository()
-            >>> modular_pipelines.has_modular_pipeline("__root__")
-            True
-            >>> modular_pipelines.has_modular_pipeline("doesnt exist")
-            False
-        """
-        return modular_pipeline_id in self.tree
-
-    def as_dict(self) -> Dict[str, ModularPipelineNode]:
-        """Return the repository as a dictionary."""
-        return self.tree
+"""`kedro_viz.data_access.repositories.modular_pipelines`

```

Comment:
No idea, why github is not ignoring this since it's exact same !!

Reply:
Mysterious! Maybe pulling `main` in will fix it?

Reply:
Glad I'm not the only going crazy. I was looking through the blames and couldn't tell what changed!